lib/tools/adb-commands.js

```diff
@@ -1301,7 +1304,8 @@ methods.killProcessByPID = async function killProcessByPID (pid) {
       }
       log.info(`Cannot kill PID ${pid} due to insufficient permissions. Retrying as root`);
       try {
-        becameRoot = await this.root();
+        let {isSuccessful} = await this.root();
```

Comment:
const

Reply:
That doesn't work, it can't override the previous definition of `let becameRoot = false`.